CODE_STYLE.md

```diff
@@ -5,17 +5,15 @@ considerations.
 
 ## Linters
 
-We use `gometalinter` to run a number of linters, the exact list can be found
-in [linter.json](linter.json). Some of these are slow and expensive to run, but
-a subset can be found in [linter-fast.json](linter-fast.json) that run quickly
-enough to be run as part of an IDE.
+We use `golangci-lint` to run a number of linters, the exact list can be found under linters
+in [.golangci.yml](.golangci.yml). [Installation](https://github.com/golangci/golangci-lint#install) and [Editor Integration](https://github.com/golangci/golangci-lint#editor-integration) for it can be found in the readme of golangci-lint.
 
 For rare cases where a linter is giving a spurious warning, it can be disabled
 for that line or statement using a [comment directive](https://github.com/alecthomas/gometalinter#comment-directives), e.g.
```

Comment:
The `nolint` section in the golangci-lint manual: https://github.com/golangci/golangci-lint#nolint

Reply:
whoops, fixed it now